cmd/lora/main.go

```diff
@@ -115,7 +119,24 @@ func main() {
 		}, []string{"method"}),
 	)
 
-	go subscribeToLoRaBroker(svc, mqttConn, logger)
+	msub, err := mqttsub.NewSubscriber(cfg.loraMsgURL, cfg.subTimeout, logger)
+	if err != nil {
+		logger.Error(fmt.Sprintf("Failed to create MQTT subscriber: %s", err))
+		os.Exit(1)
+	}
+
+	go msub.Subscribe(loraServerTopic, func(msg messaging.Message) error {
+		m := lora.Message{}
+		if err := json.Unmarshal(msg.Payload, &m); err != nil {
+			logger.Warn(fmt.Sprintf("Failed to Unmarshal message: %s", err.Error()))
+			return err
+		}
+		if err := svc.Publish(context.Background(), "", m); err != nil {
+			return err
+		}
+		return nil
+	})
```

Comment:
I would keep this in a separate method like it was previously done with `subscribeToLoRaBroker`

Reply:
@manuio that might demand passing subscriber as a param, and I do not think it will be more elegant.

@blokovi please check how it is done in other services and align.

Reply:
@drasko subscribers and publishers are passing as a params, so we can do that also here